frappe/utils/data.py

```diff
@@ -1947,6 +1947,7 @@ def sql_like(value: str, pattern: str) -> bool:
 	"None": lambda a, b: a is None,
 	"like": sql_like,
 	"not like": lambda a, b: not sql_like(a, b),
+	"is": lambda a, b: (a and (b == 'set')) or (not a and (b == 'not set')),
```

Comment:
One liners with and/or for chaining are needlessly confusing. Can you write a normal function instead?

Reply:
Also add a test somewhere in this file: https://github.com/frappe/frappe/blob/39c511a9b336162727b03bc63f8db65f45ed99e9/frappe/tests/test_utils.py#L189-L209

Reply:
thanks for you comment. its my first pr, i`m try